README.rst

```diff
@@ -1,29 +1,60 @@
-Linear Stage Component
-++++++++++++++++++++++
+##############
+ts-LinearStage
+##############
 
-This is the readme for the linear stage python api.
+The LinearStage is a CSC for the Vera C. Rubin Observatory.
+It controls a Zaber linear motor.
 
-Manual
-------
-This is the manual_ for the linear stage from Zaber_.
+Installation
+============
 
-.. _manual: https://www.zaber.com/manuals/A-LST
-.. _Zaber: https://www.zaber.com/
+.. code::
 
-Installation
+    setup -kr .
+    scons
+
+.. code::
+
+    pip install .[dev]
+    pytest --cov lsst.ts.LinearStage -ra
+
+Requirements
 ------------
-This package should be installed using the eups method.
+Run the ``develop-env`` docker image.
+
+Usage
+=====
+
+.. code::
+
+    from lsst.ts import salobj
+    linear_stage = salobj.Remote(name="LinearStage", domain=salobj.Domain(), index=1)
+
+.. code::
+
+    await linear_stage.cmd_getHome.set_start(timeout=10)
+    await linear_stage.cmd_moveAbsolute.set_start(distance=10, timeout=10)
+    await linear_stage.cmd_moveRelative.set_start(distance=10, timeout=10)
+    await linear_stage.cmd_stop.set_start(timeout=10)
+
+.. code::
+
+    position = await linear_stage.tel_position.aget()
+    print(position.position)
+
+Support
+=======
+N/A
+
 
-.. code-block:: bash
+Roadmap
```

Comment:
Roadmap??? 
I'm interested as to what you are thinking here! Could be really useful.